apps/cli/project.json

```diff
@@ -24,16 +59,6 @@
                 }
             }
         },
-        "package": {
```

Comment:
If we don't package then only the ts will end up in the npm package as build isn't part of our pipeline

Reply:
Nice catch